pidgin/app.py

```diff
@@ -228,4 +277,32 @@ def send_query(query_txt):
 
 @app.route('/_status', methods=['GET'])
 def health_check():
+    """
+    Health check endpoint
+    ---
+    tags:
+      - system
+    responses:
+      200:
+        description: Healthy
+      default:
+        description: Unhealthy
+    """
     return 'Healthy', 200
+
+
+@app.route('/swagger')
```

Comment:
idk if we should expose this as an endpoint. it seems like something that should be automated via a script or build process. what happens if you hit this endpoint when this is in production?